presto-main/src/main/java/com/facebook/presto/operator/scalar/MapTransformKeyFunction.java

```diff
@@ -189,7 +190,8 @@ private static MethodHandle generateTransformKey(Type keyType, Type transformedK
         body.append(typedSet.set(newInstance(
                 TypedSet.class,
                 constantType(binder, transformedKeyType),
-                divide(positionCount, constantInt(2)))));
+                divide(positionCount, constantInt(2)),
+                BytecodeExpressions.constantString(MAP_TRANSFORM_KEY_FUNCTION.getSignature().getName()))));
```

Comment:
Statically import `BytecodeExpressions.constantString`

In Presto code base, statical methods are usually statically imported. (Use reasonable judgement. For example,  `of`, `copyOf`, `empty`, `toString` are never statically imported. These function names are not informative when statically imported.)